app/src/main/res/layout/activity_cpr_rate.xml

```diff
@@ -13,7 +13,7 @@
             android:layout_width="129dp"
             android:layout_height="129dp"
             android:layout_marginTop="150dp"
-            app:srcCompat="@drawable/heart"
+            app:srcCompat="@drawable/cpr_rate"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
```

Comment:
There's still a warning on this image, missing contentDescription

Reply:
I have the same 

### Missing `contentDescription` attribute on image